README.md

```diff
@@ -120,6 +120,26 @@ Example:
 
 Drip does not work with STDIN. You cannot use drip for running configs which use the stdin plugin.
 
+## Building Logstash documentation
+
+To build the Logstash Reference on your local machine, clone the following repos:
+
+[logstash]https://github.com/elastic/logstash - contains main docs about core features
```

Comment:
this links need parenthesis around them [name]`(`https://...`)`

Reply:
good catch! meant to change them before pushing. :-/